src/routes/dashboard/(admin)/labs/QuotaForm.svelte

```diff
@@ -47,22 +66,47 @@
         <tr>
           <th>Laboratory</th>
           <th class="table-cell-fit">Quota ({total})</th>
+          <th class="table-cell-fit">Delete/Restore</th>
         </tr>
       </thead>
       <tbody>
-        {#each labs as { id, name, quota } (id)}
+        {#each labs as { id, name, quota, deletedAt } (id)}
           {@const placeholder = quota.toString()}
           <tr>
-            <td class="!align-middle">{name}</td>
+            <td class="!align-middle">
+              {#if deletedAt === null}
+                {name}
+              {:else}
+                <strike>
+                  {name}
+                </strike>
```

Comment:
Just an aesthetic nitpick...
```suggestion
                <strike>{name}</strike>
```